README.md

```diff
@@ -10,19 +10,17 @@ To achieve this, we want everyone to understand what is expected of them, the th
 
 
 ## Table of Contents
- - [About IDF](/company/README.md)
- - [About our platform](/platform/README.md)
- - [Roles](/roles/README.md)
-     - [Developer](/roles/developer/README.md)
- - [Workflows](/workflows/README.md)
-    - [Credentials](/workflows/credentials/README.md)
-    - [Team-trips](/workflows/team-trips/README.md)
+ - [Company](/company/README.md)
+ - [Guides](/guides/README.md)
+ - [Library](/library/README.md)
```

Comment:
Note that in prev. implementation it was a great starting point for a developer:
![image](https://user-images.githubusercontent.com/5278175/46072142-47db4480-c18a-11e8-9a7b-5bd628f2a981.png)


In this implementation we use only a single hierarchy level and it's not very easy to find an information you need